packages/angular/projects/ui-angular/src/setup.jest.ts

```diff
@@ -5,14 +5,7 @@ Object.defineProperty(window, 'CSS', { value: null });
 Object.defineProperty(document, 'doctype', {
   value: '<!DOCTYPE html>',
 });
-Object.defineProperty(window, 'getComputedStyle', {
```

Comment:
Removing it because this was blocking `dom-testing-library` from finding accessible components. I confirmed that CI passes as expected without it.